rundeckapp/grails-app/controllers/rundeck/interceptors/ProjectSelectInterceptor.groovy

```diff
@@ -70,11 +71,13 @@ class ProjectSelectInterceptor {
         if(controllerName=='menu' && ( actionName in ['home'] )){
             return true
         }
+        if(controllerName=='error' && ( actionName in ['fiveHundred'] )){
```

Comment:
Instead of adding this hack, can we exclude the interceptor from processing this controller-action tuple? It seems to be logical, but I don't know about the implications.

Reply:
We can add the controller+action to the excluded actions here: https://github.com/rundeck/rundeck/blob/fc2e59be849479b33864dd1b6b12d0c4f9d18e66/rundeckapp/grails-app/controllers/rundeck/interceptors/ProjectSelectInterceptor.groovy#L55

Honestly, we would be doing exactly the same thing with the same result.

Reply:
I addressed the changes requested to exclude the interceptor if error request